config/develop/namespaced/glue-workflow.yaml

```diff
@@ -4,12 +4,17 @@ stack_name: "{{ stack_group_config.namespace }}-glue-workflow"
 dependencies:
   - develop/namespaced/glue-tables.yaml
   - develop/namespaced/glue-job-S3ToJsonS3.yaml
+  - develop/namespaced/glue-job-JSONToParquet.yaml
+  - develop/namespaced/glue-job-compare-parquet.yaml
```

Comment:
Just to confirm that it won't be an issue if we rename this and the S3ToJsonS3 to `glue-job-python-shell.yaml` and it just reduces the complexity.

Reply:
We are renaming the template: `/recover/templates/glue-job-S3ToJsonS3.yaml` to `glue-job-python-shell.yaml` but we will still have the separate stack configs (in this example):
- develop/namespaced/glue-job-JSONToParquet.yaml
- develop/namespaced/glue-job-compare-parquet.yaml

And they will update the [template path they are pointing to here](https://github.com/Sage-Bionetworks/recover/blob/main/config/develop/namespaced/glue-job-S3ToJsonS3.yaml#L2) to the renamed `glue-job-python-shell.yaml` under templates. Hope that alleviates your concern!

Reply:
That's perfect.